### ISSUE TITLE
Audio Issues

### ISSUE THREAD

**Issue by traynor1987** [unknownskl/xbox-xcloud-client#346]
Video seems to work fine, but I get audio crackling most of the time. Would you be able to add the option of the end user changing the audio bit rate, mine needs to be higher bit rate on steam deck for some reason, other will prob be able to use whatever you have currently set, video is spot on.

Not related but I had the same issue when trying to stream the ps5 as well (audio wise).

Otherwise Amazing Work :)

**Comment by trashbus99** [comment]
what's your wifi like? I did too until I switched to ethernet

**Comment by traynor1987** [comment]
Xbox is connected to the router via ethernet, but steam deck obviously using wifi. i set the reset sound to back l4 button so over minute i press it to fix it, hopefully can be fixed in the future.

**Comment by trashbus99** [comment]
yeah..i noticed in wifi on my laptop it desynced after a a while..lag was bad.
